.ncurc.yaml

```diff
@@ -1,7 +1,5 @@
 ## Add a TODO comment indicating the reason for each rejected dependency upgrade added to this list, and what should be done to resolve it (i.e. handle it through a story, etc).
 reject: [
-  # TODO: v6+ (ref: https://github.com/sindresorhus/get-port/releases/tag/v6.0.0) is an ESM library and thus not compatible with CommonJS. Future story needed to resolve.
-  "get-port",
```

Comment:
Does this library now support both ESM and CommonJS?

Reply:
Still only ESM but with a little workaround ESM's can be used in CommonJS.